worker/worker.go

```diff
@@ -415,21 +415,21 @@ func (w *Worker) slabMigrateHandler(jc jape.Context) {
 		}
 	}
 
-	// migrate the slab
-	numShardsMigrated, surchargeApplied, err := w.migrate(ctx, slab, up.ContractSet, dlContracts, ulContracts, up.CurrentHeight)
+	// migrate the slab and handle alerts
+	err = w.migrate(ctx, slab, up.ContractSet, dlContracts, ulContracts, up.CurrentHeight)
 	if err != nil {
-		jc.Encode(api.MigrateSlabResponse{
-			NumShardsMigrated: numShardsMigrated,
-			SurchargeApplied:  surchargeApplied,
-			Error:             err.Error(),
-		})
-		return
+		var objects []api.ObjectMetadata
+		if res, err := w.bus.ListObjects(ctx, api.ListObjectOptions{SlabKey: &slab.Key}); err != nil {
+			w.logger.Errorf("failed to list objects for slab key; %w", err)
+		} else {
+			objects = res.Objects
+		}
+		w.alerts.RegisterAlert(ctx, newMigrationFailedAlert(slab.Key, slab.Health, objects, err))
```

Comment:
Seems like we are no longer considering whether the slab wasn't found before registering the alert.

Reply:
This was done on purpose but your comment made me double check and I made a mistake. I was under the impression this flow never returned that error but I missed the error in `UpdateSlab`. I remember we added it but I figured it became redundant over time somehow.